lib/sequent/migrations/versions.rb

```diff
@@ -64,7 +69,11 @@ def self.start_offline!(new_version)
           fail MigrationDone if current_migration.status.nil?
           fail ConcurrentMigration if current_migration.status != MIGRATE_ONLINE_FINISHED
 
-          current_migration.update(status: MIGRATE_OFFLINE_RUNNING)
+          current_migration.update(
+            status: MIGRATE_OFFLINE_RUNNING,
+            target_projectors:,
+            target_records:,
```

Comment:
Maybe sort the arrays before updating, makes it slightly easier to read/debug the migration table later.